content/en/releases/version-skew-policy.md

```diff
@@ -239,3 +239,13 @@ or **{{< skew currentVersionAddMinor -3 >}}**)
 Running a cluster with `kube-proxy` instances that are persistently three minor versions behind
 `kube-apiserver` means they must be upgraded before the control plane can be upgraded.
 {{</ warning >}}
+
+## Controller clients and kube-apiserver upgrades
```

Comment:
thanks for the PR, this is being discussed in api-machinery right now, will want to fold in the outcomes there into the updates here

/sig api-machinery
cc @MikeSpreitzer @deads2k 

/hold

Reply:
Okay

Reply:
Hi @divya-mohan0209, there’s a hold placed on the PR from @liggitt.